.gitignore

```diff
@@ -55,3 +55,5 @@ profile-*
 # lock files
 yarn.lock
 package-lock.json
+
+!test/fixtures/eval/node_modules
```

Comment:
This is the correct solution to getting the fixture into the repo. Adding a binary archive would not be acceptable in my opinion.